### ISSUE TITLE
GPS nav makes phone hot and eats battery when plugged in.

### ISSUE THREAD

**Issue by lbdroid** [TeamApexQ/apexq#11]
Did a trip through the mountains out west, about 2 hours. Copilot GPS (free) with $10 3d nav upgrade (makes it effectively the premium version, but with US AND Canada maps instead of just US). Started out with a full battery, plugged in to rental car's USB power supply, which was power/charge only, no idea what current it supplies. At the end of two hours, the phone was fairly warm and the battery was down to about half.

So basically, the phone, while running copilot, was drawing more power than what was being supplied by the USB charger. Way too much.

**Comment by infernix** [comment]
I have had that in the past on other phone and car chargers that only supply 500mAh. 1000mAh car chargers work fine. So it's not exclusive to apexq

**Comment by lbdroid** [comment]
I suspect that this issue was related to the usb charging issue. It may be fixed.

**Comment by infernix** [comment]
Is this still an issue?
